torchci/lib/bot/retryBot.ts

```diff
@@ -22,15 +23,12 @@ function retryBot(app: Probot): void {
         ctx.payload.workflow_run.head_branch !== defaultBranch
       ) ||
       attemptNumber > 1 ||
-      allowedWorkflowPrefixes.every(
-        allowedWorkflow => !workflowName.toLowerCase().includes(allowedWorkflow)
+      allowedWorkflowPrefixes[repo].every(
```

Comment:
This will crash if invoked for say `pytorch/text` repo, so one needs to add some logic and test to account for such contingency.